src/pages/team/index.tsx

```diff
@@ -94,24 +75,44 @@ const AtCapacityNotice = ({teamData}: {teamData: TeamData}) => {
           />
         </svg>
       </span>
-      <div className="ml-8 flex flex-col space-y-2 p-2">
-        <span>
-          Your team account is full. You can add more seats to your account
-          through the Stripe Billing Portal.
-        </span>
-        {subscriptionData?.portalUrl && (
-          <Link href={subscriptionData.portalUrl}>
+      {billingScheme === 'tiered' && (
```

Comment:
This second adjusts the copy and CTA of the At Capacity Notice based on the billing scheme.